queries/countries.rq

```diff
@@ -4,6 +4,10 @@ SELECT
   ?safeName
   ?description
   (SAMPLE(?website) AS ?website)
+  (SAMPLE(?banner) AS ?banner)
```

Comment:
Note to self: we need to make this sample earlier to ensure it matches the variables we get from Commons.